tools/hangar/src/paths.ts

```diff
@@ -26,6 +26,9 @@ export const targetWingConsoleAppSpec =
 export const targetWingConsoleServerSpec =
   process.env.HANGAR_WINGCONSOLE_SERVER_SPEC ??
   `file:${path.join(repoRoot, `apps/wing-console/console/server`)}`;
+export const targetWingConsoleDesignSystemSpec =
+  process.env.HANGAR_WINGCONSOLE_DESIGN_SYSTEM_SPEC ??
+  `file:${path.join(repoRoot, `apps/wing-console/console/design-system`)}`;
```

Comment:
It seems like "@wingconsole/design-system` is only consumed as a devDependency of @wingconsole/ui. Since it's not a runtime dependency, I don't think hangar needs to be concerned about installing it since it won't attempt to actually resolve it. WDYT?

Reply:
Good catch! It should actually depend on it as a dependency (not devDependency).